docs/technical/editor-integration.md

```diff
@@ -206,20 +206,32 @@ missing or lost it won't let you launch the external editor.
 
 ### Support for JetBrains Toolbox editors
 
-Now GitHub Desktop support editors installed through JetBrains Toolbox. 
-The technique used to achieve that is using `toolboxShellScriptName` parameter
-to check if, in the default section for scripts in JetBrainsm Toolbox a script
+Now GitHub Desktop support editors installed through JetBrains Toolbox.
+The technique used to achieve that is using `jetBrainsToolboxScriptName` field
+to check if, in the default section for scripts in JetBrainsm Toolbox, a script
 with the corresponding name exists.
 
-
 ```ts
 {
   name: 'JetBrains PyCharm',
   ...
-  toolboxShellScriptName: 'pycharm',
+  jetBrainsToolboxScriptName: 'pycharm',
 },
 ```
 
+**Note:** Don't use `jetBrainsToolboxScriptName` field on JetBrains community and edu
+editions. When JetBrains Toolbox generates the scripts. It doesn't consider the
```

Comment:
Small nit:
```suggestion
**Note:** Don't use `jetBrainsToolboxScriptName` field on JetBrains community and edu
editions. When JetBrains Toolbox generates the scripts, it doesn't consider the
```

Also maybe this should be reworded not to explicitly mention the community and edu editions, but to explain that, in general, Desktop can't handle multiple editions of the same JetBrains IDE installed simultaneously, since JetBrains Toolbox will use the same script name and override it for existing versions. Is that correct? Or is it really specific to community and edu editions?